package_readme.md

```diff
@@ -4,92 +4,83 @@
 
-If you are using [docker](https://docker.com) to run your code, we provide a wrapper command `wandb docker` that mounts your current directory, sets environment variables, and ensures the wandb library is installed. Training your models in docker gives you the ability to restore the exact code and environment with the `wandb restore` command.
+&nbsp;
 
-## Web Interface
+# Integrations
 
-[Sign up for a free account →](https://wandb.com)
-[![Watch the video](https://i.imgur.com/PW0Ejlc.png)](https://youtu.be/EeqhOSvNX-A)
-[Introduction video →](https://youtu.be/EeqhOSvNX-A)
+Use your favorite framework with W&B. W&B integrations make it fast and easy to set up experiment tracking and data versioning inside existing projects. For more information on how to integrate W&B with the framework of your choice, see [W&B Integrations](https://docs.wandb.ai/guides/integrations) in the W&B Developer Guide.
 
-## Detailed Usage
+&nbsp;
 
-Framework specific and detailed usage can be found in our [documentation](http://docs.wandb.com/).
+# Contribution guidelines
+Weights & Biases ❤️ open source, and we welcome contributions from the community! See the [Contribution guide](https://github.com/wandb/wandb/blob/main/CONTRIBUTING.md) for more information on the development workflow and the internals of the wandb library. For wandb bugs and feature requests, visit [GitHub Issues](https://github.com/wandb/wandb/issues) or contact support@wandb.com.
 
-## Testing
+&nbsp;
 
-To run basic test use `make test`.  More detailed information can be found at CONTRIBUTING.md.
+# Academic Researchers
+Reach out to W&B Support at support@wandb.com to get a [free academic license](https://www.wandb.com/academic) for you and your research group.
 
-We use [circleci](https://circleci.com) for CI.
+&nbsp;
 
-# Academic Researchers
-If you'd like a free academic account for your research group, [reach out to us →](https://www.wandb.com/academic)
+# W&B Community
 
-We make it easy to cite W&B in your published paper. [Learn more →](https://www.wandb.com/academic)
-[![](https://i.imgur.com/loKLiez.png)](https://www.wandb.com/academic)
+Be a part of the growing W&B Community and interact with the W&B team in our [Discord](https://wandb.me/discord). Stay connected with the latest ML updates and tutorials with [W&B Fully Connected](https://wandb.ai/fully-connected).
 
-## Community
-Got questions, feedback or want to join a community of ML engineers working on exciting projects?
+&nbsp;
 
-<a href="https://bit.ly/wb-slack"><img src="https://svgshare.com/i/M93.svg" alt="slack" width="55"/></a> Join our [slack](https://bit.ly/wb-slack) community.
+# License
 
-[![Twitter](https://img.shields.io/twitter/follow/weights_biases?style=social)](https://twitter.com/weights_biases) Follow us on [Twitter](https://twitter.com/weights_biases).
```

Comment:
oh we don't want to link our twitter anymore?

Reply:
x?